corehq/apps/analytics/ab_tests.py

```diff
@@ -50,3 +50,13 @@ def context(self):
     'live_preview_dec2016',
     (LIVE_PREVIEW_ENABLED, LIVE_PREVIEW_DISABLED)
 )
+
+NEW_USER_NUMBER_OPTION_SHOW_NUM = 'show_number'
+NEW_USER_NUMBER_OPTION_HIDE_NUM = 'hide_number'
+
+
+NEW_USER_NUMBER = ABTestConfig(
+    'New User Phone Number',
+    'new_phone_aug2016',
```

Comment:
I wonder if this id should be different? Not sure if product/sales wants this to appear as the same test or a new one...